### ISSUE TITLE
Prior profiles get delete

### ISSUE THREAD

**Issue by pkohut** [team4eng/Civil_3D#1]
For example.  Prior profiles for Carlson and Autocad/Civil3d can get deleted when running the setup scripts

**Comment by pkohut** [comment]
Fix here is to change delete only the ESD_C3D_2021 profile from the registry. 
```
REG DELETE "HKEY_CURRENT_USER\Software\Autodesk\AutoCAD\R24.0" /f
```
to
```
REG DELETE "HKEY_CURRENT_USER\Software\Autodesk\AutoCAD\R24.0\Profiles\ESD_C3D_2021" /f
```

**Comment by pkohut** [comment]
Fixed in commit c9698a8a48a85ef5742c24dce1e3ab66fc0d5456.
